Fix: toColumns mixin generated with wrong name

0biWanKenobi opened the pull request:
In `data_class_writer.dart` we expect a mixin named `${table.entityInfoName}ToColumns`, but then we create one named `${table.baseDartName}ToColumns` in `table_writer.dart`.

0biWanKenobi committed:
fix: toColumns mixin generated w/ wrong name

In `data_class_writer.dart` we expect a mixin named `${table.entityInfoName}ToColumns`, but then we create one named `${table.baseDartName}ToColumns` in `table_writer.dart.

simolus3 reviewed (commented):
Thanks! To avoid this problem in the future, could you move the name generation into a common getter?

E.g. `String get toColumnsMixin => '${entityInfoName}ToColumns'` in `DriftElementWithResultSet`. Both places could then refer to that getter instead of constructing the name themselves.

0biWanKenobi commented:
Sounds like a good idea! I'll do it asap 🫡

0biWanKenobi committed:
Merge branch 'simolus3:develop' into develop

0biWanKenobi committed:
refactor: common String for mixin name

define `toColumnsMixin` String in `DriftElementWithResultSet`
and reuse it in `DataClassWriter` and `TableWriter` to avoid
future mismatches in mixin name.

simolus3 reviewed (approved):
Thanks 👍
